OsmAnd/res/layout/favorite_save_bottom_sheet_buttons.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could we use default vertical buttons Layout, but add one button as bottomSheetItem?